problems/countWords.js

```diff
@@ -1,21 +1,29 @@
 /**
-* Takes an array of words and counts how many times each word occurs in the array.
-* 
-* @param {string[]} arr - An array of lowercase words
-* @returns {object} - an object where the key is the word, and the value is the count of that word
-*
-* ex: countWords(["good", "good", "great"]);
-* returns: { "good": 2, "great": 1 }
-* 
-* ex: countWords(["dog", "cat", "goat"]);
-* returns: { "dog": 1, "cat": 1, "goat": 1}
-* 
-* ex: countWords([]);
-* returns: {}
-*/
+ * Takes an array of words and counts how many times each word occurs in the array.
+ *
+ * @param {string[]} arr - An array of lowercase words
+ * @returns {object} - an object where the key is the word, and the value is the count of that word
+ *
+ * ex: countWords(["good", "good", "great"]);
+ * returns: { "good": 2, "great": 1 }
+ *
+ * ex: countWords(["dog", "cat", "goat"]);
+ * returns: { "dog": 1, "cat": 1, "goat": 1}
+ *
+ * ex: countWords([]);
+ * returns: {}
+ */
 
-function countWords() {
-  
-}
+const countWords = (arr) => {
+  let object = {};
+  arr.forEach((el) => {
```

Comment:
Nice job using forEach. I remember before you used a map, and I told you to use forEach. Way to use the feedback!